src/termdict/sstable_termdict/sstable/vint.rs

```diff
@@ -0,0 +1,66 @@
+const CONTINUE_BIT: u8 = 128u8;
```

Comment:
can move it to common with vint on u32

Reply:
Yeah I created this PR by mistake... I just wanted to use github to see the diff.